dali/operators/reader/loader/video/video_test.cc

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2021, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Maybe it can be named video_test_base.cc or video_test_utils.cc?

Reply:
Done